analyze-ms-stats/calc-manuscript-stats.py

```diff
@@ -0,0 +1,132 @@
+import json
+import pandas as pd
+import matplotlib
+import argparse
+import time
+from pathlib import Path
+import multiprocessing
+import subprocess
+
+def analyze_commit(commit):
+    """Access files and data in variables.json associated with each commit
+    Accepts commit ID as string
+    Returns list of 5 statistics"""
+
+    try:
+        variablesCommand = "git show " + commit + ":./variables.json"
+        variables = json.loads(subprocess.getoutput(variablesCommand))
+    except json.decoder.JSONDecodeError:
+        exit(commit + " not found")
+
+    date = variables['pandoc']['date-meta']
+    clean_date = variables['manubot']['date']
+    num_authors = len(variables['manubot']['authors'])
+    word_count = variables['manubot']['manuscript_stats']['word_count']
+
+    # Access files and data in references.json associated with each commit
+    referencesCommand = "git show " + commit + ":./references.json"
+    references = json.loads(subprocess.getoutput(referencesCommand))
```

Comment:
Thank you for catching this! I also moved the string creation outside of the try/except block, since that shouldn't fail (and definitely not with a JSON error).